src/runtime_strings.js

```diff
@@ -105,6 +105,9 @@ function UTF8ArrayToString(heapOrArray, idx, maxBytesToRead) {
  * @return {string}
  */
 function UTF8ToString(ptr, maxBytesToRead) {
+#if ASSERTIONS
+  assert(typeof ptr == 'number');
```

Comment:
Does this need to accept bigints for wasm64 as well?

Reply:
No, we decided to always treat pointers as numbers, even in wasm64.